internal/testing/backend/certifyLegal_test.go

```diff
@@ -387,15 +387,15 @@ func TestLegal(t *testing.T) {
 						Package: &model.IDorPkgInput{PackageInput: testdata.P1},
 					},
 					Legal: &model.CertifyLegalInputSpec{
-						Attribution: "Copyright Jeff",
+						Attribution: "Copyright Bob",
```

Comment:
This seems like the examples reordered. Is this expected or are the test flaky and/or order dependent?

Reply:
this is a result of how we are doing the ingestion. `Attribution` can be very long and in previous PRs we decided to remove it from the index (as it causes postgres to error). So now that we have changed it to `UpdateNewValues()` on duplicate, it will replace these values (before it was set to ignore - where it would not make any changes and just drop it).

Reply:
This was the error we had when we removed it:

```
{"level":"error","ts":1724690303.02192,"caller":"collector/collector.go:108","msg":"emit error: unable to ingest document: error assembling graphs for \"file:///../guac-data/docs/spdx/spdx_vuln.json\" : certifyLegalSrc failed with error: input: ingestCertifyLegals IngestCertifyLegals :: bulk upsert certifyLegal node: insert nodes to table \"certify_legals\": pq: index row size 2864 exceeds btree version 4 maximum 2704 for index \"certifylegal_source_id_declare_7172c32e01
```

Reply:
This was done back in: https://github.com/guacsec/guac/pull/2088

Reply:
Ooh, thank you!